Marble Blast Platinum/platinum/server/scripts/mp/gameConnection.cs

```diff
@@ -403,5 +403,31 @@
 	%hypo = VectorLen(%dist);
 	%pitch = -mAsin((getWord(%dist, 2) / %hypo) * 0.7);
 
+	return %angle TAB %pitch;
+}
+
+
+function GameConnection::pointToHighestValueNearestGem(%this) {
+	%pos = %this.player.getPosition();
+	%yp = transformToHighestValueNearestGem(%this.player.getGravityRot(), %pos);
```

Comment:
%yp here can be ""  since the function below returns nothing if the nearest doesn't exist. you'll want to check if %yp's field count is equal to 2 before setting the camera pitch/yaw below. Also would probably rather you return empty string instead of just return; as that implies void instead of a return type, to keep consistency.